Tests/FTAPIKitTests/AsyncTests.swift

```diff
@@ -2,7 +2,7 @@
 import Foundation
 import XCTest
 
-@available(macOS 12, iOS 15, watchOS 8, tvOS 15, *)
+@available(macOS 10.15, iOS 13, watchOS 6, tvOS 13, *)
```

Comment:
Tady není nutné to snižovat zpět, protože pak testy nebudou procházet ve starším Xcode.